Source/WebKit/WebProcess/Plugins/PDF/UnifiedPDF/PDFPresentationController.h

```diff
@@ -101,6 +101,7 @@ class PDFPresentationController : public ThreadSafeRefCountedAndCanMakeThreadSaf
 
     enum class DocumentAnchorPoint : uint8_t { TopLeft, Center };
     std::optional<VisiblePDFPosition> pdfPositionForCurrentView(DocumentAnchorPoint, bool preservePosition = true) const;
+    WebCore::FloatPoint anchorPointForDocument(DocumentAnchorPoint) const;
```

Comment:
It's unclear which coordinate system the return value is in (but you just moved this).

Reply:
I'll fix it before landing!

Reply:
Fixed in https://github.com/WebKit/WebKit/commit/63c15fcae74a7342e867f02b78753fb838895649